docs/index.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 entries:
   conjur-oss:
   - apiVersion: v1
-    created: "2020-06-18T16:52:36.877060888-04:00"
+    created: "2020-09-07T14:58:09.009245+03:00"
```

Comment:
@izgeri @abrahamko are these intentional changes or something that gets changed automatically?

Reply:
All the changes in the index file are the result of running the `reindex.sh` script.